pages/account_and_service_management/account_information/security-specification-pci/guide.en-gb.md

```diff
@@ -0,0 +1,127 @@
+
+To filter connections, you must set up a firewall using IPtables. 
+
+#### 2.2 Vulnerability scans
+
+You are authorized to perform vulnerability scans on the service you have subscribed to. OVHcloud doesn't have to be previously informed.
+Security measures deployed by OVHcloud (especially network protection) aren't disabled, because such an audit's purpose is to demonstrate a clear vision of the security level of the customer's infrastructure.
+You are not authorized to use your service to scan other infrastructures.
+
+### 3. Service Level Agreement (SLA)
+
+The SLA varies between 99.9% and 99.999% and differs between offers and ranges. Please refer to the specific terms and conditions of service for more details.
+
+### 4. Backups
+
+#### 4.1 Technical backups
+
+As a part of our reslience plan in the Control Plane, we perform backups to maintain our Service Level Agreement. These technical backups cannot be activated at the customer's request.
+
+
+#### 4.2 Customer data backups
+
+| **Option** | **Granularity** | **RPO** | **RTO** | **Documentation** |
+| --- | --- | --- | --- | --- |
+| Backup of an Instance | Instance | Depends on the date of the last backup and the delay of incident resolution | Depends on the size of the Instance | [Backing up an instance](/pages/public_cloud/compute/save_an_instance)<br>[Backups and restoration](/pages/public_cloud/compute/create_restore_a_virtual_server_with_a_backup)|
+
+
+### 5. Logs
+
+| **Source** | **Content** | **Documentation** |
+| --- | --- | --- |
+| Control Panel | Logs of interactions made by admin, technical or billing contacts in the Control Panel and services they have access to, using API calls. |- <https://api.ovh.com/console/#/me> (see `/me/api/logs`)<br>- [List of API calls done with your account](https://api.ovh.com/console/#/me/api/logs/self~GET)<br>- [List of API calls done on services you have access to](https://api.ovh.com/console/#/me/api/logs/services~GET)<br>- [Get your audit logs](https://api.ovh.com/console/#/me/logs/audit~GET) |
+
+### 6. API
+
+| **Name** | **Capacity** | **Documentation** |
+| --- | --- | --- |
+| Control Panel and service | Manage customer accounts and services on which each account has access rights. | [Preparing an environment for using the OpenStack API](/pages/public_cloud/compute/prepare_the_environment_for_using_the_openstack_api)<br>[Getting started with the OpenStack API](/pages/public_cloud/compute/starting_with_nova)<br>[API Rate Limits](/pages/public_cloud/compute/api_rate_limits) |
+
+### 7. Accounts - User
+
+#### 7.1 Control plane
+
+Using your customer account on the OVHcloud Control Panel, you are able to manage your service using [three different contacts](/pages/account_and_service_management/account_information/managing_contacts).<br>
+OVHcloud uses another account with an internal NIC to refer a customer having subscribed to several services.
```

Comment:
Hello, @H4516022Y I do not know if internal NIC is an appropriate term to use here. I am not sure to understand the statement.

Reply:
hello, Yes it is and it's for internal use to identify any customer having several NICs and services